_pages/about-us/teams/products-and-platforms.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: Products and Platforms
 ---
-Products and Platforms is one of the 5 business units within 18F. We strive to incubate tools for digital government.
```

Comment:
How about: _We help our agency partners conceptualize and build products for digital government_. (?) 

(I wanted to add in the bit about agency partners to stress that we're not building tools of our own accord/for our own purposes; rather, we're working w/others in government to help them strengthen their organizations and achieve their unique missions.)